### PR TITLE
DLP: added sample for inspect with hotword rule

### PR DESCRIPTION
**Pull request by dinesh-crest**
DLP: added sample for inspect with hotword rule
Added unit test cases for same

reference: https://cloud.google.com/dlp/docs/creating-custom-infotypes-likelihood.md#match-mrn

### CODE REVIEW

File: dlp/inspectWithHotwordRules.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+'use strict';
+
+// sample-metadata:
+//  title: Inspects strings
+//  description: Inspect a string using the Data Loss Prevention API.
+//  usage: node inspectWithHotwordRules.js my-project string minLikelihood maxFindings infoTypes customInfoTypes includeQuote hotwordRegexPattern
+
+function main(
+  projectId,
+  string,
```
Comment (realjordanna) [inline dlp/inspectWithHotwordRules.js:24]: textToInspect?

File: dlp/inspectWithHotwordRules.js [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+  [infoTypes, customInfoTypes] = transformCLI(infoTypes, customInfoTypes);
+  // [START dlp_inspect_hotword_rule]
+  // Imports the Google Cloud Data Loss Prevention library
+  const DLP = require('@google-cloud/dlp');
+
+  // Instantiates a client
+  const dlp = new DLP.DlpServiceClient();
+
+  // The project ID to run the API call under
+  // const projectId = 'my-project';
```
Comment (realjordanna) [inline dlp/inspectWithHotwordRules.js:41]: why do you have these things commented out when you pass them in via variables?

**Review by realjordanna — CHANGES_REQUESTED** [review]

File: dlp/inspectWithHotwordRules.js (hunk 1 above)
Comment (dinesh-crest) [reply dlp/inspectWithHotwordRules.js:24]: @realjordanna will keep it as a string to keep things consistent with other existing samples. Will update my previous PRs if I have used textToInspect. Thanks for pointing this out.

File: dlp/inspectWithHotwordRules.js (hunk 2 above)
Comment (dinesh-crest) [reply dlp/inspectWithHotwordRules.js:41]: The parameters will not be shown to users reading the docs. The commented lines will help them know what things they need to define before running the sample.

**Review by kweinmeister — APPROVED** [review]

**Pull request merged**